package.json

```diff
@@ -244,10 +244,10 @@
                 "description": "The location of the generated JavaScript code (the bundle file). Normally this should be \"${workspaceRoot}/.vscode/.react\"",
                 "default": null
               },
-              "iosRelativeProjectPath": {
```

Comment:
Let's make a soft deprecation and remove this gradually. I suggest to keep this option for now but change its description to notice that it's being deprecated and will be removed and suggest using new `runArgs` option, in the next release we will remove if from launch.json and in next after that - remove it completely from the code.

What do you think?

Reply:
Please see https://github.com/Microsoft/vscode-react-native/pull/517/files#diff-a617f538cd4cda30458661c4f3924240R51

Reply:
Yeah, I saw that. I just meant that we might want to keep this option available for launch.json but change it's description to emphasize its deprecation